projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -2803,3 +2815,18 @@ function wpcom_launchpad_get_latest_draft_id() {
 
 	return $cached_draft_id;
 }
+
+/**
+ * Checks if the current site primary domain is a WPCOM domain.
+ *
+ * @return bool Will return true if the primary domain is a WPCOM domain.
+ */
+function wpcom_launchpad_is_primary_domain_wpcom() {
+	if ( ! ( new Automattic\Jetpack\Status\Host() )->is_wpcom_platform() ) {
+		return false;
```

Comment:
We won't get past this check, because this check will always be executed on atomic. We need to check if there's another way to get the site slug in the Atomic context.

Reply:
This check will return true on simple and Atomic sites. So it should pass on AT. Reference: https://github.com/Automattic/jetpack/blob/5ff2353c4e9819e0ac32ffc0b28c31b3c494b207/projects/packages/status/src/class-host.php#L77